### ISSUE TITLE
Composer dependency missing

### ISSUE THREAD

**Issue by silverbackdan** [Kocal/SymfonyMailerTesting#16]
I needed to add a dependency manually:
`composer req --dev psr/http-factory`

Otherwise the error appears:
```
Cannot determine controller argument for "Kocal\SymfonyMailerTesting\Controller\MailerController::reset()": the $responseFactory argument is type-hinted with the non-existent class or interface: "Psr\Http\Message\ResponseFactoryInterface". 
```

This should be added to composer dependencies.

Nice package, thank you!

**Comment by Kocal** [comment]
Hi, sorry for not seeing your issue before!

In the readme I suggest installing `nyholm/psr7 when using the library with Cypress, but I suppose the controller auto-wiring will fails when no PSR-7 implementation is installed.

I suppose you are using Behat, so you don't really need PSR-7 implementation in fact. 

I will see to remove controllers from container when no PSR-7 implementation is available. WDYT?

**Comment by silverbackdan** [comment]
Hi @Kocal - no issue at all. That's right I'm implementing with Behat and have no need for the controllers so I think that sounds like a good solution to just remove that dependency requirement unless Cypress is being used.

**Comment by Kocal** [comment]
That will be fixed by #20
